pkg/maintainer/maintainer.go

```diff
@@ -0,0 +1,27 @@
+package maintainer
+
+import (
+	"context"
+
+	"github.com/keep-network/keep-core/pkg/bitcoin"
+)
+
+func Initialize(
+	ctx context.Context,
+	config Config,
+	btcChain bitcoin.Chain,
+	chain RelayChain,
+) {
+	if config.Relay {
+		go newRelay(ctx, btcChain, chain)
+	}
+
+	// TODO: Check if launching multiple maintainers is really beneficial.
+	//       Panic on one maintainer goroutine will crush the whole program.
+	//       Also, if we misconfigure one maintainer and it cannot launch,
+	//       should we cancel all the maintainers that already launched properly?
```

Comment:
That's the same problem as with the client handling multiple wallets 🤷🏻 

I think we should allow launching all types of maintainers at once. Even if the service crashed, the service should get restarted.

Reply:
OK.
I updated the TODO in 43106ec2d46347892753e797b45acc83a0f9eb1d.